src/sentry/static/sentry/app/views/settings/account/accountSecurity/accountSecurityDetails.jsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * AccountSecurityDetails is only displayed when user is enrolled in the 2fa method.
+ * It displays created + last used time of the 2fa method.
+ *
+ * Also displays 2fa method specific details.
+ */
+import {Box, Flex} from 'grid-emotion';
+import {withRouter} from 'react-router';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {
+  addErrorMessage,
+  addSuccessMessage,
+} from '../../../../actionCreators/settingsIndicator';
+import {t} from '../../../../locale';
+import AsyncView from '../../../asyncView';
+import Button from '../../../../components/buttons/button';
+import CircleIndicator from '../../../../components/circleIndicator';
+import DateTime from '../../../../components/dateTime';
+import RecoveryCodes from './components/recoveryCodes';
+import RemoveConfirm from './components/removeConfirm';
+import SettingsPageHeader from '../../components/settingsPageHeader';
+import TextBlock from '../../components/text/textBlock';
+import U2fEnrolledDetails from './components/u2fEnrolledDetails';
+
+const ENDPOINT = '/users/me/authenticators/';
+
+const DateLabel = styled.span`
+  font-weight: bold;
+  margin-right: 6px;
+  width: 100px;
+`;
+
+const Phone = styled.span`
+  font-weight: bold;
+  margin-left: 6px;
+`;
+
+class AuthenticatorDate extends React.Component {
+  static propTypes = {
+    label: PropTypes.string,
+    /**
+     * Can be null or a Date object.
+     * Component will return null if date is undefined, but will
+     * have value "never" if it is null
+     */
+    date: PropTypes.string,
+  };
+  render() {
+    let {label, date} = this.props;
+    // `date` can be null
+    if (typeof date === 'undefined') return null;
```

Comment:
Why wouldn't it be defined?

Reply:
Outdated case, previously this was used on detailed views when authenticator was not enrolled.